packages/integration-test-app/windows/integrationtest/TestHostHarness.cpp

```diff
@@ -85,70 +81,81 @@ void TestHostHarness::OnInstanceLoaded(const InstanceLoadedEventArgs &args) noex
       });
 }
 
-winrt::fire_and_forget TestHostHarness::StartListening() noexcept {
+IAsyncAction TestHostHarness::StartListening() noexcept {
```

Comment:
Since returning IAsyncAction, should make name clearer this loops infinitely.

Reply:
Might be worth putting this in library code instead of client code... "Live until cancel" seems like a broadly useful pattern.